my_class.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+# Represents a list of elements that pass certain conditions
+class MyList
+  include MyEnumerable
+
+  def initialize(*element)
+    @list = element
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Kindly make sure to create your own `each` method and not use the default one from the ruby language